README.md

```diff
@@ -208,6 +208,9 @@ maba_webpack:
                                         # for example, set dev_server_public_path and public_path to overwrite
                                             # http://localhost:8080/compiled/ and /compiled/
                                             # see inside your webpack.config.js for more info
+        # set location of cached manifests. Useful for deploy, when you don't want to include your cache directory
```

Comment:
Just to make clear - this is not preferred way for deployment. Or do you run `maba:webpack:compile` in one server, then warm up the cache in production?
You should also be aware that manifest can be different for each environment (or whether you use webpack-dev-server or just webpack).